src/components/Country.js

```diff
@@ -0,0 +1,52 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useLocation } from 'react-router-dom';
+import { getCities } from '../redux/citiesSlice';
+import Tile from './Tile';
+
+export default function Country() {
+  const location = useLocation();
+  const dispatch = useDispatch();
+  const { id, name } = location.state;
+
```

Comment:
- [ ] As mentioned above I would recommend using [URL parameters](https://v5.reactrouter.com/web/example/url-params) to handle data from a specific item.

Here is the error that occurs when we try to access the country route directly from the browser URL. `{ id, name }` doesn't exist as it is null
<img width="355" alt="image" src="https://user-images.githubusercontent.com/40503704/187992627-e9169598-dfaf-4287-a84b-4dec3f90fa12.png">